internal/Picker/Picker.js

```diff
@@ -559,16 +579,16 @@ const PickerBase = class extends ReactComponent {
 	};
 
 	handleDown = () => {
-		const {joined, orientation} = this.props;
+		const {changedBy, joined, orientation} = this.props;
 
-		if (joined && orientation === 'horizontal') {
+		if (joined && orientation === 'horizontal' && changedBy === 'enter') {
 			this.setIncPickerButtonPressed();
 		}
 
 		if (joined && this.pickerButtonPressed === 1) {
 			this.handleIncrement();
 
-			if (orientation === 'vertical') {
+			if (orientation === 'vertical' || (orientation === 'horizontal' && changedBy === 'leftRight')) {
```

Comment:
The condition `orientation === 'horizontal'` looks redundant.. WDYT?

Reply:
fixed it.